modules/authentication/src/handlers/local.ts

```diff
@@ -756,6 +773,45 @@ export class LocalHandlers implements IAuthenticationStrategy {
     return 'Email changed successfully';
   }
 
+  async resendVerificationEmail(
+    call: ParsedRouterRequest,
+  ): Promise<UnparsedRouterResponse> {
+    if (!this.sendVerificationEmail) return 'Configuration not set to send email';
+
+    let { email } = call.request.params;
+    if (AuthUtils.invalidEmailAddress(email)) {
+      throw new GrpcError(status.INVALID_ARGUMENT, 'Invalid email address provided');
+    }
+    email = email.toLowerCase();
+    const user: User | null = await User.getInstance().findOne({
+      email: email,
+    });
+    if (isNil(user)) throw new GrpcError(status.NOT_FOUND, 'User not found');
+    const verificationToken: Token | null = await Token.getInstance().findOne({
+      type: TokenType.VERIFICATION_TOKEN,
+      userId: user._id,
+    });
+    if (isNil(verificationToken)) {
+      throw new GrpcError(status.NOT_FOUND, 'Verification email token does not exist');
+    }
+    const serverConfig = await this.grpcSdk.config.get('router');
+    const url = serverConfig.hostUrl;
+    const result = { verificationToken, hostUrl: url };
+    const link = `${result.hostUrl}/hook/authentication/verify-email/${result.verificationToken.token}`;
+    await this.emailModule
+      .sendEmail('ChangeEmailVerification', {
+        email: email,
+        sender: 'no-reply',
+        variables: {
+          link,
+        },
+      })
+      .catch(e => {
+        ConduitGrpcSdk.Logger.error(e);
+      });
+    return 'Verification email resent';
```

Comment:
Make this `Verification code sent`.
We shouldn't treat this as a resend route/function.